js/pages/ssn.js

```diff
@@ -0,0 +1,398 @@
+
+    const template = `   
+        <div class="row ssn-content-header">   
+        </div>
+
+        <div class="row ssn-content-body">
+            <p>You have answered all of the questions in this survey. To submit your answers, select the “Submit Survey” button.</p>
+        </div>
+
+        <div class="row ssn-content-buttons">
+            <div class="col-md-3">
+                <button class="ssn-button" type="button" id="endMessageOnePrevious">Previous</button>
+            </div>
+            <div class="col-md-6">
+            </div>
+            <div class="col-md-3">
+                <button class="ssn-button" type="button" id="endMessageOneNext">Submit</button>
+            </div>
+        </div>
+    `;
+
+    content.innerHTML = template;
+
+    document.getElementById('endMessageOnePrevious').addEventListener('click', () => {
+        if (ssnNineDigits) socialSecurityNineDigits();
+        if (ssnFourDigits) socialSecurityFourDigits();
+    });
+
+    document.getElementById('endMessageOneNext').addEventListener('click', async () => {
+        
+        const formData = ssnNineDigits ? { ssnNineDigits } : { ssnFourDigits };
+
+        await storeSocial(formData);
+        location.reload();
+    });
+}
+
+const endMessageTwo = () => {
+
+    const template = `   
+        <div class="row ssn-content-header">    
+        </div>
+
+        <div class="row ssn-content-body">
+            <p>Thank you. This survey will remain available on your MyConnect dashboard in case you decide to provide this information in the future. To exit this survey, select the “Close Survey" button.</p>
+        </div>
+
+        <div class="row ssn-content-buttons">
+            <div class="col-md-3">
+                <button class="ssn-button" type="button" id="endMessageTwoPrevious">Previous</button>
+            </div>
+            <div class="col-md-6">
+            </div>
+            <div class="col-md-3">
+                <button class="ssn-button" type="button" id="endMessageTwoNext">Close</button>
+            </div>
+        </div>
+    `;
+
+    content.innerHTML = template;
+
+    document.getElementById('endMessageTwoPrevious').addEventListener('click', () => {
+        socialSecurityFourDigits();
+    });
+
+    document.getElementById('endMessageTwoNext').addEventListener('click', () => {
+        location.reload();
+    });
+}
+
+const clearError = (div) => {
+    div.innerHTML = '';
+    div.style.visibility = 'hidden';
+}
+
+const displayError = (div, message) => {
+    div.innerHTML = message;
+    div.style.visibility = 'visible';
+}
+
+const checkNineDigitValid = (input) => {
+
+    if (input.length !== 9) return false;
+
+    if (input[0] === '9' || input.slice(0, 3) === '666') return false;
+    if (input === '111111111' || input === '333333333' || input === '078051120' || input === '219099999') return false;
+    if (input.slice(0, 3) === '000' || input.slice(3, 5) === '00' || input.slice(-4) === '0000') return false;
```

Comment:
```
 if (input.slice(0, 3) === '000' || input.slice(3, 5) === '00' || input.slice(-4) === '0000') return false;
 return true;
```

It can be `return ! (input.slice(0, 3) === '000' || input.slice(3, 5) === '00' || input.slice(-4) === '0000') `

Reply:
I'm going to leave this as is so they look consistent